ledger/accountdb.go

```diff
@@ -4105,6 +4108,7 @@ func reencodeAccounts(ctx context.Context, tx *sql.Tx) (modifiedAccounts uint, e
 }
 
 // MerkleCommitter allows storing and loading merkletrie pages from a sqlite database.
+//
```

Comment:
Any idea how these got added?  Seems like some tool did it automatically?

Reply:
local go 1.19 I guess, the go fmt in each version is behaving differently like meme.

Reply:
I will take that out and tame all these formatter.

Reply:
@cce I know you looked at 1.19 formatter.  When does it do this?

Reply:
I remember there is a go 1.19 PR, which contains some lines of formatting change to comments tho, will be happy to see if we can let it stop messing with the good ASCII comment formatting.

Reply:
The change can be reflected in go 1.19 fmt PR: https://github.com/algorand/go-algorand/pull/4516/files#diff-9183c533bbd2a46eb609f432dffa31911582f30b68c8eeb02352d6f8f3e2312aR528-R531

Reply:
```suggestion
```